llvm-examples/sparse_sum_float_test.cc

```diff
@@ -6,22 +6,22 @@ using namespace taco;
 
 int main() {
     // Create formats
-    Format list({Sparse});
+    Format list({Dense});
```

Comment:
I guess this should be `sparse`
```suggestion
    Format list({Sparse});
```

Reply:
OK!